### ISSUE TITLE
Description of the role of the AB

### ISSUE THREAD

**Issue by dwsinger** [w3c/w3process#833]
The process currently [says](https://www.w3.org/2023/Process-20231103/#ab-role)

> The Advisory Board provides ongoing guidance to the Team on issues of strategy, management, legal matters, process, and conflict resolution. 

I'm not sure that now we're incorporated this is a helpful description and it risks overlap with the Board. I wonder if it would be better to describe the _scope_ of the advice given to management, rather than the _nature_ of it? Something like (this is a strawman/example):

> The Advisory Board provides ongoing guidance to the Team on issues related to the operation of the technical program and process of the Consortium.

**Comment by chrisn** [comment]
I think we should continue to include strategy, (strategy as it relates to the technical programme).

**Comment by dwsinger** [comment]
Maybe "issues" related to the technical program is the wrong word, perhaps "matters" (which would more clearly include strategy)?

**Comment by chrisn** [comment]
My concern was more about limiting the role to "operation" of the technical program, so would suggest "strategy and operation of the technical program", which should help with the overlap.

**Comment by dwsinger** [comment]
My concern is that once you start including aspects like this, you start having to enumerate them all, or there may be a perceived implication that omission of others is intentional. ("It says strategy, but not "management"…"). I would say that operation included strategy, but is there a better word that more clearly does so?

**Comment by cwilso** [comment]
I agree with @chrisn that "strategy" is important here, and "strategy and operation of the technical program" is a good way to say it.  Operation and matters do not (to me) imply strategy.

Perhaps it would be best to capture the role of the AB wrt "expressing matters of importance to the Membership" or something like that?

**Comment by chaals** [comment]
I agree with @cwilso that we should capture the idea that the AB advises the team on "what is the membership concerned about". I think the next sentence of the paragraph @dwsinger referenced already does that. (For my taste that sentence could be trimmed a bit, but IMHO it isn't broken as is, either).

I'd shorten @dwsinger's proposed new first sentence, leading to the paragraph saying

> The Advisory Board provides ongoing guidance to the Team on the technical program and process of the Consortium. The Advisory Board also serves the Members by tracking issues raised between Advisory Committee meetings, soliciting Member comments on such issues, and proposing actions to resolve these issues. The Advisory Board manages the [evolution of the Process Document](https://www.w3.org/2023/Process-20231103/#GAProcess). As part of a [W3C Council](https://www.w3.org/2023/Process-20231103/#w3c-council), members of the [Advisory Board](https://www.w3.org/2023/Process-20231103/#advisory-board) hear and adjudicate on [Submission Appeals](https://www.w3.org/2023/Process-20231103/#SubmissionNo) and [Formal Objections](https://www.w3.org/2023/Process-20231103/#FormalObjection).

**Comment by mnot** [comment]
This text from @chaals:

> The Advisory Board also serves the Members by tracking issues raised between Advisory Committee meetings, soliciting Member comments on such issues, and proposing actions to resolve these issues.

moves the AB closer to an AC leadership / chairing role. I'm not against that, just wanted to surface it for consideration.

**Comment by cwilso** [comment]
@mnot that text isn't new; it's been in the Process description of the AB for a couple of decades: https://www.w3.org/2023/Process-20231103/#AB.  (It was introduced in the 2003 Process update.)

**Comment by mnot** [comment]
Ah - weird, I didn't see that when I looked... thx

**Comment by brentzundel** [comment]
The role of the AB was recently adjusted in #1021
I believe that may address this issue and recommend closing

**Comment by brentzundel** [comment]
there is also a related discussion here: https://github.com/w3c/AB-memberonly/issues/248

**Comment by brentzundel** [comment]
No objections raised since marked "proposed to close"
Closing
